alphastats/DataSet_Preprocess.py

```diff
@@ -155,26 +158,43 @@ def _imputation(self, method: str):
         )
         self.preprocessing_info.update({"Imputation": method})
 
+    def _linear_normalization(self, array):
+        """Normalize data using l2 norm without breaking when encoutering nones
+        l2 = sqrt(sum(x**2))
+
+        Args:
+            array (pd.Series): array to normalize (1D array)
+
+        Returns:
+            np.array: normalized array
+        """
+        square_sum_per_row = array.pow(2).sum(axis=1, skipna=True)
```

Comment:
does `axis=1` work if you pass a Series?

Reply:
Good catch, I changed the code and didn't change the docstring when I went from Series to Dataframes, fixed